### DISCUSSION TITLE
WebDataset

### DISCUSSION THREAD

**Discussion by usuyama** [open-mmlab/mmselfsup#590] [General]
Is anyone using WebDataset with mmselfsup?

I suppose I can define a custom dataset following the [doc](https://github.com/open-mmlab/mmselfsup/blob/master/docs/en/tutorials/1_new_dataset.md), but just wondering if anyone using wds already.

I've been using webdataset for a few projects, and so far happy with the format and scalability. For example, [open_clip](https://github.com/mlfoundations/open_clip#data) uses webdataset for image-text datasets, but wds can be used for other setups like image-only or other annotations in json files.

**Comment by fangyixiao18** [comment]
Hi, if you implemented a custom dataset of WebDataset, welcome to create a PR, currently we don't have such feature.
Thanks~
